build_scripts/android/build.sh

```diff
@@ -16,11 +16,11 @@ if [[ ! -d "${sourcepath}" ]]; then
     exit 2
 fi
 
-if [[ "${stl}" == "c++" || "${stl}" == "gnustl" ]]; then
+if [[ "${stl}" == "c++" ]]; then
```

Comment:
Do we want to just get rid of the ability to specify stl, since there is only one option anyway?

Reply:
Sure, let's give it a try. I also decided to remove STL from the Android library path, will run packaging to make sure everything ends up in the right place.